docs/console.html

```diff
@@ -0,0 +1,804 @@
+                </figcaption>
+                <figure>
+                    <figcaption>The code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.log("Node count: %d, and the time is %f.", document.childNodes.length, Date.now());
+</code></pre>
+                </figure>
+                <figure>
+                    <img class="console-files/format-substitution.png" alt="Example subsitution output" />
+                    <figcaption>
+                        The output of the previous code sample.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="formatting-dom-elements-as-javascript-objects">Formatting DOM elements as JavaScript objects</h3>
+
+            <p>
+                When you log a DOM element to the console it is displayed in an XML format.
+                This is the same way this are displayed in the Elements panel.
+                To display the JavaScript representation you may either use the <code>dir()</code> method or the object format specifier in a <code>log()</code> statement.
+            </p>
+
+            <figure>
+                <figcaption>The different views for logging an element.</figcaption>
+                <figure>
+                    <img src="console-files/log-element.png" alt="Element logged" />
+                    <figcaption>Default <code>log()</code> view of an element.</figcaption>
+                </figure>
+
+                <figure>
+                    <img src="console-files/dir-element.png" alt="Element logged using dir()" />
+                    <figcaption>Element logged using the <code>dir()</code> method.</figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="styling-console-output-with-css">Styling console output with CSS</h3>
+            <p>
+                The CSS format specifier allows you to customize the display in the console.
+                Simply have the specifier at the start of the string and give the style you wish to apply as the second parameter.
+            </p>
+
+            <figure>
+                <figcaption>Styling log information</figcaption>
+                <figure>
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
+console.log("%cThis will be formatted with large, blue text", "color: blue; font-size: x-large");                       
+</code></pre>
+                </figure>
+                <figure>
+                    <img src="console-files/format-string.png" alt="Formatted string" />
+                    <figcaption>Example code output.</figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="measuring-how-long-something-takes">Measuring how long something takes</h3>
```

Comment:
Updated the document to reflect this. Used the existing sample code and added an arrow pointing out the annotation. GH isn't detecting the diff though, so this comment stays open.